### ISSUE TITLE
Could not get results from After Effects. Ensure that Preferences > General > Allow Scripts to Write ...

### ISSUE THREAD

**Issue by pelallemant** [BenZed/after-effects#28]
Hi,

I figured out that in some cases, if the response contains \n or \r characters, it is not escaped. So the js files are broken because strings are only on 1 line. 

Do you have the time to investigate ?

Thanks

**Comment by BenZed** [comment]
Is this on a Mac or PC?

**Comment by pelallemant** [comment]
The bug occurs under windows. For example when exporting the content of a TextDocument with a "\n" will systematically break the code by adding a line.

**Comment by BenZed** [comment]
I'll take a look, thanks for reporting!

**Comment by pelallemant** [comment]
Another example is some exception: if you include a file and it is not present, then i get the same error than in the title.

The following code returns me the same error, instead of an exception with trace:

```
$.evalFile(File('ImNotPresent.jsx'));
```

**Comment by esoper** [comment]
I'm getting the same issue on mine unfortunately. I am trying to get this to work with v15.0.0 (Build 180)

**Comment by MattBanak** [comment]
Was messing with this last night on Windows, seems to be a sync issue. Stepping through slowly allows time for AE to write the contents of the result file and this error doesn't happen.

I suspect that there's an issue with the execute function for windows (platform-win.js:66)

Seems to me that the program just executes the script then resolves the promise without any checks or waiting.

**Comment by MattBanak** [comment]
Yeah, adding a 1 second timeout before resolving fixed the issue for me, submitting a PR later.

Admittedly this is a band-aid fix but should be good enough for most people. Ideally we would do some sort of watching for the file to exist. Might PR that solution instead.

**Comment by alexabunny** [comment]
I found that this issue happens if AfterEffects is already open before `ae.execute` is called. Letting `ae` open AE, run the script, and close AE gets around this issue for me.
